### PR TITLE
New api

### LINKED ISSUE #23: Clean up pull request for `new-api`
**Description** 

There is a larger PR open to change the API for Flottform to allow easier changes to the UI. We want to merge this idea, fix open issues with the API and then optimize it.

**Links / References**

- See PR #22 

**Tasks / Definition of Done**

- [ ] Remove prototype API files
- [ ] Merge `new-api` PR
- [ ] Add tasks for open issues after the merge

**Further information**

After prototyping on Flottform and making sure the idea works for different use-cases, we need to clean up and find a good API for developers to use.

### PR DESCRIPTION
**Pull request by TamaraBogantseva**
**Checklist**

- [x] Resolves #23 
- [x] Labelled and assigned a reviewer

**Reviewer**

- [ ] I've checked out the code and tested it myself.

**Changes**

- _Add new API_ 
- _Remove old files_

### CODE REVIEW

File: flottform/forms/src/connect-to-flottform.ts
Comment (TamaraBogantseva) [inline flottform/forms/src/connect-to-flottform.ts:1]: This is an old API

File: flottform/forms/src/create-flottform-input.ts
Comment (TamaraBogantseva) [inline flottform/forms/src/create-flottform-input.ts:1]: This is an old API

File: flottform/forms/src/flottform-channel-host.ts [hunk 1]
```diff
@@ -0,0 +1,292 @@
+import { toDataURL } from 'qrcode';
+import { Logger } from '../dist';
+import { FlottformState } from '../dist/internal';
```
Comment (Narigo) [inline flottform/forms/src/flottform-channel-host.ts:3]: `dist` looks wrong to me?

Comment (TamaraBogantseva) [reply flottform/forms/src/flottform-channel-host.ts]: Yes, it should be imported from './internal' 👍🏼

**Comment by TamaraBogantseva** [comment]
We have discuss it during our sprint meeting and we can merge it now

**Pull request merged**